src/i18n/locales/en.json

```diff
@@ -1 +1,73 @@
-{}
+{
+  "blog.authors.list.title.by": "by ",
```

Comment:
From my experience - translation values should not contain spaces like this

Reply:
Interesting, is there any real reason why? I believe in some cases it makes sense, but some cases are complex where arbitrary code happens. If you check where they're used, it might make sense. (As I didn't want to refactor the utility itself that is used for the list of authors)

Reply:
Usually it produces bugs in case translation team will accidentally remove spaces in other translation file and words will be merged all together in UI. If we need to have space there reliably - it better to have it code/css than in translation file